code/game/objects/items/storage/belt.dm

```diff
@@ -619,6 +620,37 @@
 			return FALSE
 	return TRUE
 
+/obj/item/storage/belt/gun/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/weapon/gun/pistol) || !reload_on)
+		return ..()
+	var/obj/item/weapon/gun/pistol/gun = I
+	if(gun.current_mag)
+		return ..()
```

Comment:
probably want to to return
And a to chat to say that it has already a mag would be nice.
Also, if it's tactictal reload, should it not remove the mag already in place?

Reply:
ah right, tactical means theres a mag. Ill change that.